### PR TITLE
Updated Layer 1 types YANG and JSON

### LINKED ISSUE #96: Update JSON file in Appendix A
The line wrapping of the JSON file should be based on RFC 8792

OLD

`========== NOTE: '\' line wrapping per BCP XXX (RFC XXXX) ==========`

NEW

`=============== NOTE: '\' line wrapping per RFC 8792 ===============`

- [x] Upload the JSON file (not folded) on github
- [x] Re-run the RFC 8792 script
- [x] Upload the folded JSON file on github

### LINKED ISSUE #98: Container for OTN label range
It may be useful to group all the attributes used to define the OTN label range within a YANG container:

OLD
```
   +--rw range-type?      otn-label-range-type
   +--rw tsg?             identityref
   +--rw odu-type-list*   identityref
   +--rw priority?        uint8

```

NEW
```
   +--rw otn-label-range
      +--rw range-type?      otn-label-range-type
      +--rw tsg?             identityref
      +--rw odu-type-list*   identityref
      +--rw priority?        uint8

```

### LINKED ISSUE #99: Container for OTN label hop
It may be useful to group all the attributes used to define the OTN label range within a YANG container:

OLD
```
   +--:(otn)
      +--rw otn-tpn?   otn-tpn
      +--rw tsg?       identityref
      +--rw ts-list?   string

```

NEW
```
   +--:(otn)
      +--rw otn
         +--rw otn-tpn?   otn-tpn
         +--rw tsg?       identityref
         +--rw ts-list?   string

```

### PR DESCRIPTION
**Pull request by italobusi**
Updated layer1-types YANG:
- Added container for otn label hop: fix #99
- Fixed some issues with line length exceeding the 69 characters
- Added container for otn-link-bandwidth, otn-label-range-info, otn-label-start-end and otn-label-step: fix #98 

Uploaded JSON examples: fix #96 :
- Uploaded odu-label-restrictions-examples.json file
- Updated with the new conventions for comments in JSON files
- Created odu-label-restrictions-examples.txt by running rfcfold script

JSON examples aligned with the latest YANG model changes

### CODE REVIEW

**Review by haomianzheng — APPROVED** [review]

File: YANG/ccamp/layer1-types/ietf-layer1-types.yang [hunk 1]
```diff
@@ -1101,38 +1117,42 @@ module ietf-layer1-types {
...
-        "Tributary Port Number.";
-      reference
-        "RFC7139: GMPLS Signaling Extensions for Control of Evolving
-         G.709 Optical Transport Networks.";
-    }
-    leaf tsg {
-      type identityref {
-        base tributary-slot-granularity;
+        "Label hop for OTN.";
+      leaf otn-tpn {
```
Comment (agva123) [inline YANG/ccamp/layer1-types/ietf-layer1-types.yang:1124]: If we add an "otn" container then here "otn-" prefix is really unnecessary. Suggest changing it to just "tpn"

File: YANG/ccamp/layer1-types/ietf-layer1-types.yang [hunk 2]
```diff
@@ -1145,40 +1165,48 @@ module ietf-layer1-types {
...
-             Tributary Port Number.";
-          reference
-            "RFC7139: GMPLS Signaling Extensions for Control of 
-             Evolving G.709 Optical Transport Networks.";
+        "Label step for OTN";
+      choice range-type {
+        description
+          "OTN label range type, either TPN range or TS range";
+        case trib-port {
+          leaf otn-tpn {
```
Comment (agva123) [inline YANG/ccamp/layer1-types/ietf-layer1-types.yang:1178]: If we add an "otn" container then here "otn-" prefix is really unnecessary. Suggest changing "otn-tpn" to just "tpn"

File: YANG/ccamp/layer1-types/ietf-layer1-types.yang [hunk 3]
```diff
@@ -1145,40 +1165,48 @@ module ietf-layer1-types {
...
-          } 
-          type otn-ts; 
-          description
-            "Label step which represents possible increments for 
-             Tributary Slot Number.";
-          reference
-            "RFC7139: GMPLS Signaling Extensions for Control of
-             Evolving G.709 Optical Transport Networks.";
+        case trib-slot {
+          leaf otn-ts {
```
Comment (agva123) [inline YANG/ccamp/layer1-types/ietf-layer1-types.yang:1195]: If we add an "otn" container then here "otn-" prefix is really unnecessary. Suggest changing "otn-ts" to just "trib-slot"

File: YANG/ccamp/layer1-types/ietf-layer1-types.yang (hunk 1 above)
Comment (sergiobelotti) [reply YANG/ccamp/layer1-types/ietf-layer1-types.yang:1124]: I agree on the comment above, please delete "otn" up front to leaf "otn-tpn"

File: YANG/ccamp/layer1-types/ietf-layer1-types.yang (hunk 2 above)
Comment (sergiobelotti) [reply YANG/ccamp/layer1-types/ietf-layer1-types.yang:1178]: I agree on the proposed modification

File: YANG/ccamp/layer1-types/ietf-layer1-types.yang (hunk 3 above)
Comment (sergiobelotti) [reply YANG/ccamp/layer1-types/ietf-layer1-types.yang:1195]: As soon as there is a description e.g. tpn ...is tributary port number I do not think a longer name is needed.

File: YANG/ccamp/layer1-types/odu-label-restrictions-examples.json [hunk 4]
```diff
@@ -0,0 +1,263 @@
+{
+  "// examples of label-restrictions for different OTN Links": [
+    {
+      "// example": "HO-ODU1 or OTU1 Link",
+      "label-restrictions": {
+        "label-restriction": [
+          {
+            "index ": 1,
+            "// default restriction": "inclusive",
+            "otn-label-range": {
```
[the remark this answers is not in the document]
Comment (italobusi) [reply YANG/ccamp/layer1-types/odu-label-restrictions-examples.json:10]: It has been added with this PR:

https://github.com/haomianzheng/IETF-ACTN-YANG-Model/blob/3e5f6c22d88f876a9f097f6dd24a7df3428353cd/YANG/ccamp/layer1-types/ietf-layer1-types.yang#L1005-L1016

File: YANG/ccamp/layer1-types/ietf-layer1-types.yang (hunk 1 above)
Comment (italobusi) [reply YANG/ccamp/layer1-types/ietf-layer1-types.yang]: Addressed in https://github.com/haomianzheng/IETF-ACTN-YANG-Model/pull/104/commits/b80b3a8f5fbf26efc4b237f06b859a03e133b297

File: YANG/ccamp/layer1-types/ietf-layer1-types.yang (hunk 2 above)
Comment (italobusi) [reply YANG/ccamp/layer1-types/ietf-layer1-types.yang]: Addressed in https://github.com/haomianzheng/IETF-ACTN-YANG-Model/pull/104/commits/b80b3a8f5fbf26efc4b237f06b859a03e133b297

File: YANG/ccamp/layer1-types/ietf-layer1-types.yang (hunk 3 above)
Comment (italobusi) [reply YANG/ccamp/layer1-types/ietf-layer1-types.yang]: Addressed in https://github.com/haomianzheng/IETF-ACTN-YANG-Model/pull/104/commits/b80b3a8f5fbf26efc4b237f06b859a03e133b297

**Pull request merged**